src/Components/ExpandControl/ExpandControl.tsx

```diff
@@ -17,8 +17,11 @@ export class ExpandControl extends React.Component<ExpandControlProps> {
     public readonly context: ExpandContext;
 
     public render() {
+        const { state, expandId, triggerEvent } = this.props;
+        const newState = state !== undefined ? state : !this.context.isExpanded(expandId);
```

Comment:
if state in props equals true, it will always changing expand state to true?

Reply:
Yes

Reply:
By default this prop is undefined and state will change alternately